build/ps-entrypoint.sh

```diff
@@ -288,10 +294,10 @@ if [ "$1" = 'mysqld' -a -z "$wantHelp" ]; then
 			CREATE USER 'operator'@'${MYSQL_ROOT_HOST}' IDENTIFIED BY '${OPERATOR_ADMIN_PASSWORD}' ;
 			GRANT ALL ON *.* TO 'operator'@'${MYSQL_ROOT_HOST}' WITH GRANT OPTION ;
 
-			CREATE USER 'xtrabackup'@'localhost' IDENTIFIED BY '${XTRABACKUP_PASSWORD}';
-			GRANT SYSTEM_USER, BACKUP_ADMIN, PROCESS, RELOAD, LOCK TABLES, REPLICATION CLIENT ON *.* TO 'xtrabackup'@'localhost';
-			GRANT SELECT ON performance_schema.log_status TO 'xtrabackup'@'localhost';
-			GRANT SELECT ON performance_schema.keyring_component_status TO 'xtrabackup'@'localhost';
+			CREATE USER 'xtrabackup'@'%' IDENTIFIED BY '${XTRABACKUP_PASSWORD}';
```

Comment:
I am not sure that we need to have separate user. Maybe we can use existing one

Reply:
By existing one you mean `root` or `operator`? I believe we should have a `xtrabackup` user as in PXC